src/SOS/SOS.UnitTests/SOS.cs

```diff
@@ -91,6 +103,10 @@ public async Task GCTests(TestConfiguration config)
     {
         // Live only
         SkipIfArm(config);
+        if (SOSRunner.IsAlpine())
```

Comment:
Because it is currently only needed by the SOS tests. Putting it in MemberData (TestConfiguration) means putting in Microsoft.Diagnostic.TestHelpers and exposing it to everybody.  We can do that when some other tests need it.